src/components/TodoListItem.js

```diff
@@ -6,10 +6,10 @@ const TodoListItem = ({ todo, onRemoveTodo }) => {
 
     return (
         <li className={style.ListItem}>
-            {todo.fields.Title}&nbsp;
             <button type='button' onClick={() => onRemoveTodo(todo.id)}>
                 <span class="material-icons">delete_forever</span>
```

Comment:
`class` should be renamed to JSX equivalent `className`

see warning:

![Screen Shot 2022-01-16 at 5 08 17 PM](https://user-images.githubusercontent.com/10986971/149679992-97d2c12c-aa38-44ab-908f-b7c11de8fdec.png)
`